dashboard/src/features/tutorial/reducers.js

```diff
@@ -33,8 +50,22 @@ export const userInputs = (state = { accounts: [] }, action) => {
   return state
 }
 
-export default combineReducers({
-  step,
-  isShowing,
-  userInputs
-})
+export default (state = {}, action) => {
+  const tutorialRoute = state.route
+  const tutorialUserInputs = state.userInputs
+
+  delete state.currentStep // combineReducers logs error because currentStep set outside of function
```

Comment:
Can you clarify what's going on here? Deleting items out of the old state object in a reducer shouldn't do anything. `newState` should completely replace this value.